crypto_box.js

```diff
@@ -15,8 +16,15 @@ const crypto_box_BOXZEROBYTES = 16
 const crypto_box_SEALBYTES = 48
 const crypto_box_SEEDBYTES = 32
 const crypto_box_BEFORENMBYTES = 32
+const crypto_box_MACBYTES = 16
+
+const crypto_box_curve25519xsalsa20poly1305_MACBYTES = 16
+
+const crypto_box_MESSAGEBYTES_MAX = crypto_stream_xsalsa20_MESSAGEBYTES_MAX - crypto_box_curve25519xsalsa20poly1305_MACBYTES
```

Comment:
Should this have just been `crypto_stream_xsalsa20_MESSAGEBYTES_MAX - 16`? When in doubt I've erred toward copying exactly what libsodium is doing.

Reply:
This is perfect, better for clarity to explicitly call constants